specification/communication/data-plane/Microsoft.CommunicationServicesIdentity/preview/2021-03-31-preview/CommunicationIdentity.json

```diff
@@ -0,0 +1,118 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CommunicationIdentityClient",
+    "description": "Azure Communication Identity Service",
+    "version": "2021-03-31-preview"
+  },
+  "paths": {
+    "/m365identities/:issueAccessToken": {
+      "post": {
+        "tags": [
+          "TeamsToken"
+        ],
+        "summary": "Generate a new M365 Skypetoken in exchange for ACS AAD token for an identity.",
+        "operationId": "CommunicationIdentity_IssueTeamsToken",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "schema": {
+              "$ref": "#/definitions/TeamsAccessTokenRequest"
+            }
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "../../../Microsoft.CommunicationServicesCommon/stable/2021-03-07/common.json#/definitions/CommunicationErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/CommunicationIdentityAccessToken"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "TeamsAccessTokenRequest": {
+      "required": [
+        "token"
+      ],
+      "type": "object",
+      "properties": {
+        "token": {
+          "description": "The token acquired via MSAL.",
```

Comment:
Instead of saying where is it acquired from, it is better to explain what it is so the developers would know when to use this.

Reply:
Updated with suggestion "Azure Active Directory access token to acquire Skypetoken."